### PR TITLE
Fix broken link in governance.md

### PR DESCRIPTION
**Pull request by RichiH**
Signed-off-by: Richard Hartmann <richih@richih.org>

### CODE REVIEW

**Review by marefr — APPROVED** [review]
LGTM

**Review by roidelapluie — APPROVED** [review]

**Review by roidelapluie — CHANGES_REQUESTED** [review]
This PR is not correct. [maintainers.md] is defined at the bottom of the file.

**Comment by roidelapluie** [comment]
I think you are confusing: `[MAINTAINERS.md](MAINTAINERS.md)` and `[MAINTAINERS.md][maintainers.md]`.

**Comment by roidelapluie** [comment]
I guess an improvement would be to change `[maintainers.md]` by `[maintainers-md]` to avoid future confusion.

File: GOVERNANCE.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ This document describes the rules and governance of the project. It is meant to
 
 * **Team members**: Any members of the private [grafana-team][team] Google group.
 
-* **Maintainers**: Maintainers lead an individual project or parts thereof ([`MAINTAINERS.md`][maintainers.md]).
+* **Maintainers**: Maintainers lead an individual project or parts thereof ([`MAINTAINERS.md`][MAINTAINERS.md]).
```
Comment (marefr) [inline GOVERNANCE.md:7]:
```suggestion
* **Maintainers**: Maintainers lead an individual project or parts thereof ([`MAINTAINERS.md`](https://github.com/grafana/grafana/blob/master/MAINTAINERS.md)).
```

**Review by marefr — CHANGES_REQUESTED** [review]
Oops was too fast with the review. Added a suggestion, but not sure if there are other clever ways to link to other md-files in GitHub?

File: GOVERNANCE.md (hunk 1 above)
Comment (roidelapluie) [reply GOVERNANCE.md:7]: the link points to [maintainers.md]: https://github.com/search?l=&q=org%3Agrafana+filename%3AMAINTAINERS.md+path%3A%2F&ref=advsearch&type=Code&utf8=%E2%9C%93
198

File: GOVERNANCE.md [hunk 2]
```diff
@@ -194,6 +194,6 @@ If needed, we reserve the right to publicly announce removal.
 
 [coc]: https://github.com/grafana/grafana/blob/master/CODE_OF_CONDUCT.md
 [devs]: https://groups.google.com/forum/#!forum/grafana-developers
-[maintainers.md]: https://github.com/search?l=&q=org%3Agrafana+filename%3AMAINTAINERS.md+path%3A%2F&ref=advsearch&type=Code&utf8=%E2%9C%93
+[maintainers]: https://github.com/grafana/grafana/blob/master/MAINTAINERS.md
```
Comment (roidelapluie) [inline GOVERNANCE.md:197]: Shouldn't we add a maintainers.md to the other repositories as well? and then the old link would still be valid?

**Review by roidelapluie — APPROVED** [review]

**Review by marefr — APPROVED** [review]
LGTM

**Pull request merged**